internal/util/bundleutil/bundleutil.go

```diff
@@ -200,7 +201,11 @@ func (meta *BundleMetaData) BuildBundleImage(tag string) error {
 		// TODO(varsha): Make this more user friendly by accepting a template which
 		// can executed in each bundle subdirectory.
 		log.Infof("Using the specified command to build image %s", img)
-		cmd := exec.Command(meta.BuildCommand, img)
+		commandArg := strings.Split(meta.BuildCommand, " ")
+
+		// append the rest of the arguments to the command
+		commandArg = append(commandArg, "-f", filepath.Join(meta.BundleDir, "bundle.Dockerfile"), "-t", img, ".")
```

Comment:
I was under the impression that the build command string is completely owned by the user, so that they can override any args for example. In that case, why force file name, tag flag, and context?

Reply:
I was thinking of the same, but then if thats the case then user can build only image for only a single version. That is, the command would have to be:
`operator-sdk pkgman-to-bundle packagemanifests --image-tag-base quay.io/example/memcached --build-cmd "/usr/local/bin/docker build -f bundles/bundle-0.0.1/bundle.Dockerfile -t"`. If we let the user pass the path to docker file, then it needs to get executed only in a single directory.

Reply:
If we configure the docker file, then the user can pass the image tag and context. But that would need separate flags. Taking it as a single string would not be a valid docker command.